navigation/ca_move_base/launch/move_base_flex.launch

```diff
@@ -0,0 +1,56 @@
+<launch>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns"        value="create$(arg robot_id)" doc="Namespace of the robot. By default: create1"/>
+  <arg name="tf_prefix" value="$(arg ns)_tf"          doc="Tf prefix"/>
+
+  <arg name="laser" default="$(optenv LASER rplidar)" doc="Laser for mapping the environment"/>
+
+  <arg name="global_planner" value="$(optenv GLOBAL_PLANNER gbp)"/>
+  <arg name="local_planner"  value="$(optenv LOCAL_PLANNER teb)"/>
+
+  <node pkg="mbf_costmap_nav" type="mbf_costmap_nav" respawn="false" name="move_base_flex" output="screen">
+    <rosparam file="$(find ca_move_base)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
+    <rosparam file="$(find ca_move_base)/config/global_costmap_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/planners.yaml" command="load" />
+    <rosparam file="$(find ca_move_base)/config/$(arg global_planner)_global_planner_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
+    <rosparam file="$(find ca_move_base)/config/local_costmap_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/controllers.yaml" command="load" />
+    <rosparam file="$(find ca_move_base)/config/$(arg local_planner)_local_planner_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/recovery_behaviors.yaml" command="load" />
+
+    <!-- We want a single map for multi-robots -->
+    <remap from="map" to="/map"/>
+    <remap from="/laser/scan" to="/$(arg ns)/$(arg laser)/scan"/>
```

Comment:
All the topics have `(arg ns)`. What happen if you remove it and add `ns="(arg ns)"` in the node itself?